pkg/server/event_handler_test.go

```diff
@@ -201,74 +220,100 @@ contacts:
   email:
   - test.gmail.com
 `),
-			processors:       []Processor[*structpb.Struct]{&successProcessor{}},
-			successMessenger: &NoopMessenger{},
+			githubResourceBytes: getFakeMetadata(),
+			processors:          []Processor[*structpb.Struct]{&successProcessor{}},
+			successMessenger:    &NoopMessenger{},
 		},
 		{
 			name: "failed_send_downstream",
 			notification: pubsub.Message{
-				Attributes: map[string]string{"bucketId": "foo", "objectId": "bar"},
+				Attributes: map[string]string{"bucketId": "foo", "objectId": "bar", "git-commit": "test-git-commit"},
```

Comment:
why do we need "git-commit" in the attributes?

Reply:
Removed. Forgot to remove those.